src/components/main/DownloadButton.tsx

```diff
@@ -37,14 +39,16 @@ export const DownloadButton = ({ id, name }: Props): JSX.Element => {
     downloadItem({ id });
   };
   return (
-    <Button
-      handleDownload={handleDownload}
-      isLoading={isDownloading}
-      title={translateBuilder(BUILDER.DOWNLOAD_ITEM_BUTTON)}
-      ariaLabel={translateBuilder(BUILDER.DOWNLOAD_ITEM_BUTTON)}
-      loaderColor="primary"
-      loaderSize={10}
-    />
+    <span id={`${DOWNLOAD_BUTTON_ID}${id}`}>
```

Comment:
To help you could convert this to a function that takes the itemId as input and returns the string to target the element:
```ts
export const buildDownloadButtonId = (itemId: string): string => `download-button-id-${itemId}`;
```
For example. This would exist in the selectors file so you can re-use it in the tests and be sure that you have constructed your id in the same way.